src/NServiceBus.Testing.Tests/SagaTests.cs

```diff
@@ -22,6 +22,22 @@ public void MySaga()
         }
 
         [Test]
+        public void MySagaWithConstructorDependency()
+        {
+            var instance = new MySagaWithConstructorDependency(null);
```

Comment:
I think it would be nicer if the passed dependency wouldn't be `null`. You could easily change the dependency from an interface to a class and new that up.